kotlinpoet/src/test/java/com/squareup/kotlinpoet/UtilTest.kt

```diff
@@ -89,11 +89,47 @@ class UtilTest {
     assertThat("".isIdentifier).isFalse()
   }
 
+  @OptIn(ExperimentalStdlibApi::class)
   @Test fun escapeNonJavaIdentifiers() {
     assertThat("8startWithNumber".escapeIfNecessary()).isEqualTo("`8startWithNumber`")
     assertThat("with-hyphen".escapeIfNecessary()).isEqualTo("`with-hyphen`")
-    assertThat("with space".escapeIfNecessary()).isEqualTo("`with space`")
+    assertThat("with space".escapeIfNecessary()).isEqualTo("`with·space`")
     assertThat("with_unicode_punctuation\u2026".escapeIfNecessary()).isEqualTo("`with_unicode_punctuation\u2026`")
+
+    val generated = FileSpec.builder("a", "b")
```

Comment:
I suggest moving this into a separate test method under `FileSpecTest`, tests inside `UtilTest` should only be testing utilities.

Reply:
I would still like this to be in a separate test method in a different file.